assets/question_catalog/definition.json

```
@@ -4224,19 +4224,31 @@
                "highway": "elevator",
                "door:width": false
             },
+            "osm_element": ["Node", "ClosedWay"],
             "!parent": [
-               {
-                 "child": [
-                   {
-                     "osm_tags": {
-                       "indoor": "door"
-                     }
-                   }
-                 ]
-               }
-             ]
-         }
-      ]
+              {
+                "osm_element": ["OpenWay", "ClosedWay"],
```

I think it makes sense to only look for an `OpenWay` here.

```suggestion
                "osm_element": "OpenWay",
```